File: QRStockMateSL/QRStockMate.AplicationCore/Entities/Warehouse.cs
namespace QRStockMate.AplicationCore.Entities {
	public class Warehouse {
		public int Id { get; set; }
		public string Name { get; set; }
		public string Location { get; set; }
		public string Organization { get; set; }
		public int IdAdministrator { get; set; }
		public string IdItems { get; set; }
		public string Url { get; set; }
		public double Latitude { get; set; }  // Propiedad para almacenar la latitud
		public double Longitude { get; set; } // Propiedad para almacenar la longitud
	}
}

File: QRStockMateWeb/src/app/interfaces/warehouse.ts
export interface Warehouse {
    id: number;
    name: string;
    location: string;
    organization: string;
    idAdministrator: number;
    idItems: string;
    url: string;
    latitude: number;  // Propiedad para almacenar la latitud
    longitude: number; // Propiedad para almacenar la longitud
}

